coordinator/tributary/src/tendermint/mod.rs

```diff
@@ -275,9 +275,14 @@ pub struct TendermintNetwork<D: Db, T: TransactionTrait, P: P2p> {
 
 pub const BLOCK_PROCESSING_TIME: u32 = 999;
 pub const LATENCY_TIME: u32 = 1667;
-// TODO: Add test asserting this
 pub const TARGET_BLOCK_TIME: u32 = BLOCK_PROCESSING_TIME + (3 * LATENCY_TIME);
 
+#[test]
+fn assert_target_block_time_no_overflow() {
```

Comment:
This isn't about checking it doesn't overflow. It's about checking that the actual target block time *from the `tendermint-machine` crate* is equal to this.

Reply:
yea was not sure either:D 👍